core/src/test/java/hudson/FilePathTest.java

```diff
@@ -464,7 +464,7 @@ private void checkTarUntarRoundTrip(String filePrefix, long fileSize) throws Exc
     }
 
     @Test public void symlinkInTar() throws Exception {
-        if (Functions.isWindows())  return; // can't test on Windows
+        assumeTrue(!Functions.isWindows()); // can't test on Windows
```

Comment:
Then the test will be *skipped* in JUnit (and Jenkins), while before it was considered successful even though it in fact did not run.